test/networkd-test.py

```diff
@@ -457,6 +458,18 @@ def do_test(self, coldplug=True, ipv6=False, extra_opts='',
         else:
             self.fail('nameserver 192.168.5.1 not found in ' + RESOLV_CONF)
 
+        if netlabel and selinux.is_selinux_enabled():
```

Comment:
Could you split out about adding tests into another commit? And instead of networkd-test.py, please update test-network/systemd-networkd-tests.py. networkd-test.py is not maintained.

Reply:
Split commits. I moved the tests over to systemd-networkd-tests.py and also added tests for `IPv6AcceptRA.NetLabel=` and `DHCPPrefixDelegation.NetLabel=`.